build-system/pr-check/performance-tests.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview
+ * This script runs performance tests.
+ * This is run during the CI stage = test; job = performance tests.
+ */
+
+const colors = require('ansi-colors');
+const {
+  downloadDistOutput,
+  printChangeSummary,
+  startTimer,
+  stopTimer,
+  timedExecOrDie: timedExecOrDieBase,
+} = require('./utils');
+const {determineBuildTargets} = require('./build-targets');
+const {isTravisPullRequestBuild} = require('../common/travis');
+
+const FILENAME = 'performance-tests.js';
+const FILELOGPREFIX = colors.bold(colors.yellow(`${FILENAME}:`));
+const timedExecOrDie = (cmd) => timedExecOrDieBase(cmd, FILENAME);
+
+async function main() {
+  const startTime = startTimer(FILENAME, FILENAME);
+
+  if (!isTravisPullRequestBuild()) {
+    downloadDistOutput(FILENAME);
+    timedExecOrDie('gulp update-packages');
+    timedExecOrDie('gulp performance --nobuild --quiet');
+  } else {
+    printChangeSummary(FILENAME);
+    const buildTargets = determineBuildTargets(FILENAME);
+    if (buildTargets.has('RUNTIME') || buildTargets.has('FLAG_CONFIG')) {
```

Comment:
This should include a new `PERFORMANCE_TESTS` target that detects changes to `build-system/tasks/performance/**` and `build-system/pr-check/performance-tests.js`

Similar to the `E2E_TESTS` logic:

https://github.com/ampproject/amphtml/blob/6409019e2116006dad24e2c2f3897a65f8d35679/build-system/pr-check/e2e-tests.js#L49-L64

Reply:
This is misleading, we won't be running this on PRs so we don't need a new target.

Reply:
Gotcha. In that case, I'll retract this suggestion.

However, there is the risk that a breaking change to `build-system/tasks/performance/**` or `build-system/pr-check/performance-tests.js` won't be caught by PR checks and will directly break `master`, but I don't suppose there's a good way to prevent it via this PR. Punting SGTM.